docs/code-of-conduct.md

```diff
@@ -61,7 +61,7 @@ representative at an online or offline event.
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
 reported to the community leaders responsible for enforcement at
-[INSERT CONTACT METHOD].
+jj-abuse@googlegroups.com.
```

Comment:
Should we also provide the individual community leaders' names and email addresses? That would help if someone doesn't trust that the right people are behind the list alias, or if they have a problem with one community leaders.

Reply:
My hope was to have one place to maintain the active list of leaders and to make it easy for anyone to reach those leaders. I want to avoid creating too many options for the users (especially because the source document doesn't provide any such options).

The text that follows says:

> All complaints will be reviewed and investigated promptly and fairly.
> 
> All community leaders are obligated to respect the privacy and security of the
reporter of any incident.

I believe by having more than one person on that list and honoring that text, we're taking on the responsibility of ensuring that the list is adequately staffed and will handle the situation fairly.

I'd consider any failure to rise to our documented responsibilities to be a bug.

I'm thinking let's keep it simple for now and perhaps iterate on it later if needed. WDYT?

Reply:
Maybe continuing to mention the fallback to opensource@ can help with the "if they have a problem with one community leaders" case?

Reply:
I think my impression that we should list each individual here was based on the old version of the code of conduct, which has this:

> Reports should be directed to [PROJECT STEWARD NAME(s) AND EMAIL(s)], the Project Steward(s) for [PROJECT NAME]

Since that's not there anymore, maybe they decided that a single contact endpoint is enough.

> Maybe continuing to mention the fallback to opensource@ can help with the "if they have a problem with one community leaders" case?

That might still be a good idea.

Reply:
Listing the individual emails still makes sense to me, as a backup route. What if you have a problem with one of the members of the `jj-abuse` group? And, currently, it doesn't look like it's possible to view the members that are in the group, so you have no way of knowing if your email is going to a problematic person or not.

Reply:
Since I want to avoid the non-atomic duplication of effort, I've just bypassed the group and list the emails in the doc and removed the group. How's that?

Reply:
LGTM!